local-site.yml

```diff
@@ -0,0 +1,21 @@
+site:
+  title: OpenNMS.js Documentation
+  url: https://docs.opennms.com/opennms-js/
+  start_page: opennms-js::index.adoc
+content:
+  sources:
+  - url: .
+    branches: HEAD
+    start_path: docs-src
```

Comment:
We started with ALEC and used in our projects `docs` as the directory. I followed this path when I've setup HELM, PRIS, and the Horizon Core. Would be the last chance here to make it consistent :D

Reply:
I'm a bit confused what you mean by this.  Do you mean the docs should be in the `docs` directory, rather than `docs-src`?  I think originally I had done that because it conflicted with some of the other autogenerated docs I have, but now I'm not sure that's necessary anymore.

Reply:
OH!  Now I remember.  Since Antora/asciidoc can't handle API docs (like javadoc/typedoc) I want to publish the typedoc to github pages, which only supports serving either the root of the pages repo, or the `docs/` directory.  I meant to put _that_ there, not the antora output.  :)